docker/hooks/build

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Who is using this hook? dockerhub?

Reply:
Assuming you are building this in the same manner as the IoT Agents in **Docker Cloud.**  the`SOURCE_BRANCH` **Env** variable will be available. The `build` file will provide additional `org.opencontainers` labels. 

See: https://github.com/telefonicaid/iotagent-ul/issues/317#issuecomment-446630979

Reply:
NTC